chia/_tests/util/test_replace_str_to_bytes.py

```diff
@@ -57,9 +58,15 @@
     MAX_GENERATOR_REF_LIST_SIZE=uint32(512),
     POOL_SUB_SLOT_ITERS=uint64(37600000000),
     HARD_FORK_HEIGHT=uint32(5496000),
+    HARD_FORK2_HEIGHT=uint32(0xFFFFFFFF),
     PLOT_FILTER_128_HEIGHT=uint32(10542000),
     PLOT_FILTER_64_HEIGHT=uint32(15592000),
     PLOT_FILTER_32_HEIGHT=uint32(20643000),
+    PLOT_DIFFICULTY_4_HEIGHT=uint32(0xFFFFFFFF),
+    PLOT_DIFFICULTY_5_HEIGHT=uint32(0xFFFFFFFF),
+    PLOT_DIFFICULTY_6_HEIGHT=uint32(0xFFFFFFFF),
```

Comment:
what are these for ?

Reply:
This is a new concept for proof-of-space v2. My understanding is that it's a little bit like the plot filter, but perhaps more granular. Over time, we increase the difficulty, to account for hardware improving efficiency to grind. So, this is a schedule of block heights of when it's increased.